### PR TITLE
Adds skub to the game

### PR DESCRIPTION
**Pull request by lzimann**
Sprites are shit

Skub

It looks worst in game
:cl: Lzimann
:rscadd: Skub added to the game
/:cl:

### CODE REVIEW

**Comment by Shapsy** [comment]
fuck skub

**Comment by Archie700** [comment]
yeah fuck skub

**Comment by Zilenan91** [comment]
fuck YOU skub hater

**Comment by Armhulen** [comment]
Made an account just to support skub 2k16

**Comment by Archie700** [comment]
>failed check

anti-skub wins

**Comment by lzimann** [comment]
@Archie700 no that just means i'm stupid

**Comment by Archie700** [comment]
@lzimann pretty much shows what pro-skub players are

**Comment by KingDairy** [comment]
is it valid to kill pro skub players as non-antag?

**Comment by Armhulen** [comment]
Aaaand I'm no longer a pro skub player

**Comment by optimumtact** [comment]
PRO SKUB

ANTI SKUB GET THE HELL OUT

**Comment by Zilenan91** [comment]
PREACH

**Comment by Supermichael777** [comment]
Izimann combine our powers into one ultimate pull by merging the pr to your branch.

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -27,6 +27,18 @@
 	required_reagents = list("orangejuice" = 1, "cornoil" = 1)
 	result_amount = 2
 
+/datum/chemical_reaction/skub
+	name = "skub"
+	id = "skub"
+	result = null
+	required_reagents = list("spraytan" = 1, "liquidgibs" = 1)
```
Comment (KorPhaeron) [inline code/modules/reagents/chemistry/recipes/others.dm]: I don't like the idea of explaining what skub is, just make the clown spawn with it

Comment (Supermichael777) [inline code/modules/reagents/chemistry/recipes/others.dm]: it wont last past April fools  anyway

Comment (KorPhaeron) [inline code/modules/reagents/chemistry/recipes/others.dm]: This fits my design vision though

**Comment by Fox-McCloud** [comment]
This is retarded.

**Comment by PKPenguin321** [comment]
>This is retarded.

Hey fuck you anti-skubber

**Comment by KazeEspada** [comment]
I missed a meme.

File: code/game/objects/items/weapons/miscellaneous.dm [hunk 2]
```diff
@@ -10,3 +10,10 @@
 	throw_range = 5
 	w_class = 2
 	attack_verb = list("warned", "cautioned", "smashed")
+
+/obj/item/weapon/skub
+	desc = "A skub"
+	name = "skub"
+	icon = 'icons/obj/items.dmi'
+	icon_state = "skub"
+	w_class = 4
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/miscellaneous.dm]: make its attack verb skubbed.

**Comment by newfren** [comment]
DOWN WITH SKUB

**Comment by SweaterKittens** [comment]
>The anti-skub degeneracy in this PR
>Anti-skub players are the cancer that is killing /tg/

**Comment by bgobandit** [comment]
I support skub

**Comment by Ausops** [comment]
this server is a pro skub server

anti skubbers will not be tolerated

**Comment by MMMiracles** [comment]
the real question is how many people looked up what skub was before posting in this PR

**Comment by Supermichael777** [comment]
anyone who had to is obviously an pubby and an anti-skub sjw

**Comment by KazeEspada** [comment]
Dont you realise its just hair gel? Pro skubbers are trying to tear this server apart.

**Comment by optimumtact** [comment]
It's not hairgel you shitter, it's SKUB!

**Comment by KazeEspada** [comment]
Thats what pro skubs want you to think.

**Comment by Thunder12345** [comment]
Close, lock, burn the skub so it may never blight our server

**Comment by Aloraydrel** [comment]
Only old /tg/ users know this!

**Comment by optimumtact** [comment]
Newbs need to get on the skub train

**Comment by CPTANT** [comment]
Guys, I forgot if I am pro or anti skub. Can we start over?

**Comment by KazeEspada** [comment]
Who you calling newb skubby? I have over 300 confirmed kills....

**Comment by Bawhoppen** [comment]
I cannot believe in the 26th century we're still arguing about skub

**Pull request closed without merging**